proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -85,6 +85,9 @@ message PluginConfig {
   // True if the source plugin has been deleted. This dooms
   // the template.
   bool deleted = 4;
+  // True if the source plugin is inaccessible by the user.
+  // This dooms the template.
```

Comment:
```suggestion
```
I think this wording is a little weird as the template itself isn't doomed, its just the user who won't be able to use it (but can still look at it). If we ever allow people to switch out plugins (to for example a similar template of their own with the same functionality), this would also not be very forward looking.

Reply:
yes you are right 👍